### ISSUE TITLE
anbox always display in DISPLAY=:0

### ISSUE THREAD

**Issue by lukedoomer** [anbox/anbox#156]
**Description of the problem:**
anbox launch --package=org.anbox.appmgr --component=org.anbox.appmgr.AppViewActivity in DISPLAY 1 still shows org.anbox.appmgr in DISPLAY 0

**Expected:**
anbox should be showed in the specific DISPLAY

**Additional info:**
Is there an environment variable or config which we can set DISPLAY?

**Comment by morphis** [comment]
We don't support multi monitor environments where two different x-servers are running yet. You can run anbox only on one xserver at the same time. You need to start the `anbox session-manager` with the correct DISPLAY variable set to get Anbox on either one or the other screen. I guess because `anbox session-manager` is started by either a systemd or upstart job in your case it will always default to DISPLAY=:0

Setting the DISPLAY variable for `anbox launch` will not have any effect as that process will not talk to X. It will only send a dbus call to the `anbox session-manager` which then will be responsible for opening a window.

**Comment by lukedoomer** [comment]
you are right I am using arch linux and launch anbox session-manager from systemd.

**Comment by zhsj** [comment]
So what happens if you set DISPLAY env in your systemd start service? like:

```
[Unit]
Description=Anbox Session Manager

[Service]
Environment=DISPLAY=:1
ExecStart=/usr/bin/anbox session-manager

[Install]
WantedBy=default.target
```
